packages/server/package.json

```diff
@@ -50,6 +50,7 @@
   "scripts": {
     "build": "tsc -b",
     "clean": "rimraf lib *.tsbuildinfo coverage .nyc_output",
+    "generate:index": "glsp generateIndex src/browser src/common src/node -s -f",
```

Comment:
Is there a reason we are using `-s` in this repo as opposed to the dedicated index file in each sub-directory?

Reply:
Counter question: Is there a reason for dedicated index files in each subdirectory if we are generating the file anyways?
A single common entry point for the index file seems to be better in this case.
In addition, single index files in each subdirectory might tempt developers (or the TS server autoimport) to import a module from the index instead of its complete path. This can be dangerous since most of our modules are not completly isolated and importing from a (sub) index might lead to nasty circular dependency issues (especially  in combination with inversify).

In general, I would opt for a single index file at the root unless you can think of use case where sub indices are benefical.

Reply:
I'm fine with the single index file, I was just surprised about the inconsistency between the repositories.